dcpy/library/script/doe_pepmeetingurls.py

```diff
@@ -14,7 +14,7 @@ def extract_date(x):
     try:
         dt = dparser.parse(x, fuzzy=True)
         return dt.date()
-    except:
+    except:  # noqa F722
```

Comment:
This seemed like the simplest thing to do right now. I maybe should make an issue to actually abolish bare excepts in our codebase